css-cascade-5/Overview.bs

```diff
@@ -1035,6 +1035,7 @@ Cascade Sorting Order</h3>
 			For this purpose:
 
 			<ul>
+				<li>Style sheets are ordered as in <a href="https://drafts.csswg.org/cssom/#documentorshadowroot-final-css-style-sheets">final CSS style sheets</a>.
```

Comment:
Ok, sounds good to me to fixup this link in a followup.

Reply:
Will do, once this lands.